packages/gatsby-remark-images-contentful/src/index.js

```diff
@@ -91,7 +96,7 @@ module.exports = async (
   <span
     class="gatsby-resp-image-wrapper"
     style="position: relative; display: block; ${
-      options.wrapperStyle
+      options.showCaptions ? null : options.wrapperStyle
```

Comment:
should this be here? it seems unrelated

Reply:
Yes, it's unrelated to the bug, I forgot I added it with this PR, sorry about that.
I noticed this was handled differently in `gatsby-remark-images` and brought it in line with that code.

Reply:
I've removed this change, let's add it in a separate PR.